src/main/java/chess/domain/movement/BlackPawnDefaultMovement.java

```diff
@@ -0,0 +1,27 @@
+package chess.domain.movement;
+
+import chess.domain.position.Position;
+import java.util.List;
+
+public final class BlackPawnDefaultMovement implements MovementRule {
+
+    @Override
+    public boolean isMovable(Position start, Position end, boolean isAttack) {
+        int rankDifference = start.calculateRankDifference(end);
+        int fileDifference = start.calculateFileDifference(end);
+
+        boolean isEmptyAtEnd = !isAttack;
+        boolean isMatchDifference = rankDifference == -1 && fileDifference == 0;
```

Comment:
```suggestion
     boolean isMatchDifference = (rankDifference == -1 && fileDifference == 0);
```

-1과 0이 어떤 의미인지도 이름을 주어 나타내주면 좋을 것 같아요.

Reply:
다른 부분도 전부 조건문의 경우 괄호가 추가되면 좋을 것 같아요.